pkg/solidity-utils/test/foundry/UniswapV2BasePoolMathRounding.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.24;
+
+import "../../contracts/test/BasePoolMathMock.sol";
+import "../../contracts/test/UniswapV2BasePoolMathMock.sol";
+import "./utils/BasePoolMathRoundingTest.sol";
+
+contract UniswapV2BasePoolMathRoundingTest is BasePoolMathRoundingTest {
+    function setUp() public override {
+        // The delta is dependent on the implementations of computeInvariant and computeBalances. For UniswapV2, the delta is slightly higher due to the sqrt operation.
```

Comment:
Not sure why lint is not triggering an error for this, but I think we should respect the 120 char per line limit